main.tf

```diff
@@ -1,28 +1,21 @@
 resource "aws_ssm_parameter" "default" {
-  count       = "${length(var.parameters)}"
```

Comment:
Style guide recommends count be on its own line: https://www.terraform.io/docs/configuration/style.html.